src/components/Nav/Navigation.tsx

```diff
@@ -107,9 +113,9 @@ class Navigation extends React.Component<PropsType, NavigationState> {
     );
 
     return (
-      <Page header={Header} sidebar={Sidebar} onPageResize={this.onPageResize}>
+      <Page style={{ backgroundColor: 'purple' }} header={Header} sidebar={Sidebar} onPageResize={this.onPageResize}>
```

Comment:
Just for curiosity, why purple?

Reply:
Debugging purpose :) 
Was to easily see if my Page filled the whole page.

Reply:
Will remove this

Reply:
I just removed this purple color

Reply:
Ok, looks better now :)

Reply:
Purple, haha, I do pink.